DisplayInfo/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.2] - 2022-12-09
+### Syncup
```

Comment:
I understand this is sync up from Metro but pls change this to Fixed or Changed labels and add some details on the changes. Something like "Remove Nexus changes". This would help someone checking later to get a quick summary of the changes done.

Reply:
updated